src/main/java/co/inventorsoft/springresttask/service/repository/impl/EmailRepositoryImpl.java

```diff
@@ -0,0 +1,50 @@
+package co.inventorsoft.springresttask.service.repository.impl;
+
+import co.inventorsoft.springresttask.service.model.Email;
+import co.inventorsoft.springresttask.service.repository.EmailRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class EmailRepositoryImpl implements EmailRepository {
+
+    private final List<Email> emailList = new ArrayList<>();
+
+    @Override
+    public Email getEmail(int id) {
+        return emailList.stream()
+                .filter(email -> email.getId() == (id))
+                .findFirst()
+                .orElseThrow(() -> new RuntimeException("Email is not found!"));
+    }
+
+    @Override
+    public List<Email> listEmails() {
+        return new ArrayList<>(emailList);
+    }
+
+    @Override
+    public Email createEmail(Email email) {
+        emailList.add(email);
+        return email;
+    }
+
+    @Override
+    public Email updateEmail(int id, Email email) {
+        boolean isDeleted = emailList.removeIf(u -> u.getId() == (id));
```

Comment:
What is this method doing? Maybe it would be clearer if you used a stream API and updated an already saved object instead of deleting and creating a new one?